src/main/java/seedu/address/logic/commands/ArchivesCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.model.Model;
+import seedu.address.model.ModelContext;
+
+/**
+ * Lists all entries in the archives to the user.
+ */
+public class ArchivesCommand extends Command {
+
+    public static final String COMMAND_WORD = "archives";
+    public static final String COMMAND_ALIAS = "archs";
+
+    public static final String MESSAGE_SUCCESS = "Context switched to archive-context. Listed all entries in archives.";
```

Comment:
Is showing the resultant context in the success message ideal?

Reply:
In the future, I'm thinking a persistent display showing the context model is currently in. But sounds like a "nice-to-have" rather than required. So this is the bare-minimum I felt was needed for users to know the context has changed.

Reply:
I think what you have now is enough, if we have more time we can think of a better UI but we're in a bit of a squeeze.

Reply:
Right! Sounds like an enhancement for the future. We can get this feature out for v1.2 and work on enhancing it later.